test/test_tools.jl

```diff
@@ -0,0 +1,7 @@
+using Base.Test
+using Mimi
+
+@test Mimi.prettifystring("camelCaseBasic") == "Camel Case Basic"
+@test Mimi.prettifystring("camelWithAOneLetterWord") == "Camel With A One Letter Word"
+@test Mimi.prettifystring("snake_case_basic") == "Snake Case Basic"
+@test Mimi.prettifystring("_snake__case__weird_") == "Snake Case Weird"
```

Comment:
As discussed, add another test that runs the ``plot`` function and compares a pre-saved image with the new output.